CHARTER.md

```diff
@@ -20,16 +20,48 @@ The primary objectives of the resource aggregation working group includes themes
 
 ## 2. Working Group Composition
 
-The Working Group is composed of…
+This working group is composed of volunteers who have experience and interest in developer communities.
 
-The membership profile is…
+Typical members include professionals who have roles such as: developer advocate, developer relations engineer, developer evangelist, developer marketing, product management, product marketing manager, etc. 
 
+Organizations that have developer-facing tools can all benefit from a better understanding of the resources available to them. 
 
 ### 2.1 Roles & Responsibilities 
 
+To empower participation, we've identified the responsibilities for various types of roles.
+
+#### Community at Large
+
+Membership of this working group is open to all in the community. Individual one-off contributions are welcome by anybody so long as it follows the submission guidelines.
```

Comment:
```suggestion
Participation of this working group is open to all in the community. Individual one-off contributions are welcome by anybody so long as it follows the submission guidelines.
```

Reply:
LF-hosted projects typically use the word *members* to refer to the organizations sponsoring the project. Therefore, we are using *participants* in every DRF documentation instead to avoid confusion

Reply:
Good to know, thanks for the input. I updated but had to change wording slightly so added as a separate commit.